views/stats.handlebars

```diff
@@ -17,8 +17,8 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.language.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.messagestore.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.parser.min.js"></script>
-  <script type="module" src="generated/i18n.entry.js?v=v{{ version }}"></script>
-  <script type="module" src="generated/stats.entry.js?v=v{{ version }}"></script>
+  <script src="generated/i18n.entry.js?v=v{{ version }}"></script>
```

Comment:
fixing this in a separate pr but public paths need a `/` at the start of the path